package.json

```diff
@@ -44,6 +44,8 @@
   "devDependencies": {
     "babel-core": "6.6.0",
     "babel-eslint": "5.0.0",
+    "babel-istanbul": "^0.6.0",
```

Comment:
This patch should fix that in future: https://github.com/mozilla/web-ext/pull/92